<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>view_schedule_window</class>
 <widget class="QMainWindow" name="view_schedule_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Courses</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="schedule_table_widget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>811</width>
      <height>191</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Course ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Course Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Section</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Instructor</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Days</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Start Time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>End Time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Room</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="schedule_label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>200</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Shopping Cart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_button">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>340</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="show_selected_schedule_widget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>811</width>
      <height>211</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Course ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Course Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Section</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Instructor</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Days</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Start Time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>End Time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Room</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="remove_button">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>600</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enroll_button">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>630</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
